src/multi-select/tp-multi-select.ts

```diff
@@ -28,6 +31,10 @@ export class TPMultiSelectElement extends HTMLElement {
 	 * Connected callback.
 	 */
 	connectedCallback(): void {
+		this.search = this.querySelector( 'tp-multi-select-search' );
+		this.styledOptions = this.querySelectorAll( `tp-multi-select-option` );
```

Comment:
@i-am-chitti This will cause a problem when items are dynamically added / updated after the component is initialised?